packages/test/test-end-to-end-tests/src/test/detachedContainerTests.spec.ts

```diff
@@ -734,18 +734,18 @@ describeFullCompat("Detached Container", (getTestObjectProvider) => {
 		await defPromise.promise;
 	});
 
-	it("Load attached container from cache and check if they are same", async () => {
-		const container = await loader.createDetachedContainer(provider.defaultCodeDetails);
-
-		// Now attach the container and get the sub dataStore.
-		await container.attach(request);
-
-		// Create a new request url from the resolvedUrl of the first container.
-		assert(container.resolvedUrl);
-		const requestUrl2 = await provider.urlResolver.getAbsoluteUrl(container.resolvedUrl, "");
-		const container2 = await loader.resolve({ url: requestUrl2 });
-		assert.strictEqual(container, container2, "Both containers should be same");
-	});
+	// it.only("Load attached container from cache and check if they are same", async () => {
```

Comment:
Is this test failing? Why did you comment it out?

Reply:
yes it failed, the two containers aren't the same since we are not caching anymore